migrations/0005_add_questionnaire_lookup_indexes.sql

```
@@ -0,0 +1,6 @@
+-- Keep the per-question interaction path index-backed as response history grows.
+CREATE INDEX IF NOT EXISTS idx_questionnaire_sessions_user_completed_at
+  ON questionnaire_sessions (user_id, completed, completed_at);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)